vcpkg.json

```diff
@@ -54,8 +54,8 @@
         "dependencies": [
           {
             "name": "cairo",
-            "platform": "windows",
-            "default-features": false
+            "default-features": false,
+            "host": true
```

Comment:
This should probably be removed from here and put into the manifest for python3-cairosvg, then `host: true` should cascade properly.

Reply:
It looks like it's already marked as host in the manifest so it can probably be simply removed from here.